### PR TITLE
Login remember me

### PR DESCRIPTION
**Pull request by karbmk**
Implemented using sessions.
Files changed:
1. views.py

### CODE REVIEW

**Comment by nickolashe** [comment]
@karbmk @dannypernik Are we looking to set some sort of timeout for this behavior?

**Comment by karbmk** [comment]
No. As of now it has no timeout i think.

**Comment by dannypernik** [comment]
@nickolashe there's another open issue regarding timeout

File: openCurrents/views.py [hunk 1]
```diff
@@ -89,6 +89,14 @@ def get_context_data(self, **kwargs):
 
 class HomeView(SessionContextView, TemplateView):
     template_name = 'home.html'
+    def dispatch(self, *args, **kwargs):
+        try:
+            #If there is session set for profile
+            if self.request.session['profile']:
```
Comment (nickolashe) [inline openCurrents/views.py]: @karbmk i think login session is already stored in self.request.user. can't we just assert login by:
> if self.request.user.is_authenticated
?

**Comment by nickolashe** [comment]
@karbmk good idea, please the question above

File: openCurrents/views.py (hunk 1 above)
Comment (karbmk) [inline openCurrents/views.py]: But id the user has not clicked the keep me logged in button. Even then it would point to the profile which is not intended

**Comment by dannypernik** [comment]
@nickolashe your question about timeout? Looks like he's replied

File: openCurrents/views.py (hunk 1 above)
Comment (nickolashe) [inline openCurrents/views.py]: that's true, good point

**Pull request merged**